### PR TITLE
feat(testing): support positional arg for jest executor

### PR DESCRIPTION
**Pull request by MJez29**
## Current Behavior

The syntax to test a single file is different when using the jest executor versus the jest cli

```sh
jest <file>
# vs
nx test project --testFile <file>
```

## Expected Behavior

I would expect the jest executor to support a single positional argument as well

```sh
nx test project <file>
```

This is one of the common questions/complaints that I hear from developers when transitioning from our standalone apps to our monorepo

This change supports both syntaxes so that this is not a breaking change and prints a warning when both are used

### CODE REVIEW

**Comment by isaacplmann** [comment]
Run `nx documentation` locally and commit the changes to get the CI working

**Comment by MJez29** [comment]
> TypeError: stripJsonComments is not a function
I see this error now 🤔

**Comment by JamesHenry** [comment]
@MJez29 It was an issue on an older version of master, I've just pulled in the latest for you

**Comment by MJez29** [comment]
@isaacplmann @JamesHenry how can I resolve the CI failure now? It looks like a timeout to me

File: packages/jest/src/executors/jest/schema.json [hunk 1]
```diff
@@ -51,6 +51,13 @@
       "x-completion-type": "file",
       "x-completion-glob": "jest.config@(.js|.ts)"
     },
+    "_": {
+      "type": "array",
+      "description": "The name of the file to test as a single positional argument. Equivalent to `--testFile`.",
+      "items": {
+        "type": "string"
+      }
+    },
```
Comment (leosvelperez) [inline packages/jest/src/executors/jest/schema.json:60]: We don't need an extra option for this. We can specify a positional argument as the default source for any option (https://nx.dev/extending-nx/recipes/generator-options#default).

So, instead of adding this new option, you can edit the existing `testFile` to:

```diff
    "testFile": {
      "description": "The name of the file to test.",
      "type": "string",
+     "$default": {
+       "$source": "argv",
+       "index": 0
+     }
    },
```

With that in place, you get the desired behavior and don't need to handle any extra `_` option. The `testFile` option will get assigned the value of the first positional argument when the `--testFile` flag is not provided. The `--testFile` flag can still be used, in which case, it will override the positional argument if also provided.

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: docs/shared/packages/jest/jest-plugin.md [hunk 2]
```diff
@@ -161,6 +161,16 @@ To run Jest tests via nx use
 nx test frontend
 ```
 
+### Testing a Single File
```
Comment (leosvelperez) [inline docs/shared/packages/jest/jest-plugin.md:164]: I'd rephrase it a bit because the regex can match multiple files. So, the section is really showing how to test specific files.
```suggestion
### Testing Specific Files
```

**Review by leosvelperez — CHANGES_REQUESTED** [review]
Sorry, I missed this in my previous review. One last small thing left, and it should be good to go 🚀

**Review by leosvelperez — APPROVED** [review]
